airflow/config_templates/config.yml

```diff
@@ -598,6 +598,22 @@
       type: integer
       example: ~
       default: "100"
+    - name: google_oauth2_audience
+      description: The intended audience for JWT token credentials used for authorization.
+        This value must match on the client and server sides.
+        If empty, audience will not be tested.
+      type: string
+      example: project-id-random-value.apps.googleusercontent.com
+      default: ""
+    - name: google_key_path
+      description: |
+        Path to GCP Credential JSON file. If ommited, authorization based on `the Application Default
+        Credentials
+        <https://cloud.google.com/docs/authentication/production#finding_credentials_automatically>`__ will
```

Comment:
From what I see we usually do not use rst format for links in config

Reply:
It works. :-)  Probably we should update other places to use normal links.
<img width="840" alt="Screenshot 2020-07-22 at 02 07 26" src="https://user-images.githubusercontent.com/12058428/88119235-21d52e80-cbc0-11ea-99f5-3a7537b90c4c.png">